operator/controllers/btpoperator_controller.go

```diff
@@ -62,6 +62,7 @@ var (
 	ReadyStateRequeueInterval      = time.Hour * 1
 	ReadyTimeout                   = time.Minute * 1
 	RetryInterval                  = time.Second * 10
```

Comment:
How about changing `RetryInterval` var name to something like `HardDeleteCheckInterval` and passing it to `time.Sleep()` func in L661?